weasyl/errorcode.py

```diff
@@ -1,4 +1,4 @@
-# errorcode.py
+# coding=utf-8
```

Comment:
We should consider standardizing all files across the codebase to a single style when it comes to specifying file encoding. I am seeing the style you used appearing 1 time in master, ``# encoding: utf-8`` 2 times in master, and ``# -*- coding: utf-8 -*-`` 6 times in master.

On one of the tests I wrote, you suggested that file use the ``# encoding: utf-8`` variant, so perhaps standardize to that? It's the smallest of nits as the code is still functional, and I can get everything about the ads branch to work. If we are going for consistency, we should pick a style and use it.